web-registry/src/components/organisms/ProjectTopSection.tsx

```diff
@@ -290,50 +288,24 @@ function ProjectTopSection({
                       ]
                     }
                   />
-                  {metadata?.['http://regen.network/projectActivity']?.[
-                    'http://schema.org/name'
-                  ] && (
-                    <ProjectTopLink
-                      label="project activity"
-                      name={
-                        metadata?.['http://regen.network/projectActivity']?.[
-                          'http://schema.org/name'
-                        ]
-                      }
-                      url={
-                        metadata?.['http://regen.network/projectActivity']?.[
-                          'http://schema.org/url'
-                        ]?.['@value']
-                      }
-                    />
-                  )}
-                  <ProjectTopLink
-                    label="offset generation method"
-                    name={
-                      creditClassVersion.metadata?.[
-                        'http://regen.network/offsetGenerationMethod'
-                      ]
-                    }
-                  />
                 </>
               )}
-              {methodologyVersion && (
+              {!isVcsProject && (
```

Comment:
we should just move the "OFFSET GENERATION METHOD" to the additional metadata section in any case (even on existing pages like wilmot or kasigau)

Reply:
Resolving this for now per our discussion with @erikalogie yesterday, since we only want to move this to the additional metadata section for VCS projects